ci-operator/config/openshift/assisted-service/openshift-assisted-service-ocm-2.3.yaml

```diff
@@ -211,7 +211,6 @@ tests:
         ADDITIONAL_PARAMS="--with-static-network-config"
         ISO_IMAGE_TYPE=minimal-iso
         KUBE_API=yes
-        OPENSHIFT_VERSION=4.8
```

Comment:
is the PR on the service backported to ocm-2.3?
or we don't need to

Reply:
I guess not. So it would take 4.7 probably... and it would fail... Good catch.
So I think it should be backported. As OCM should have 4.8 as the only version anyway

Reply:
ocm-2.3 changes were removed since the `default` field wasn't backported over there.